### PR TITLE
LHC-729 redesign nav bar

### PR DESCRIPTION
**Pull request by rdai10**
### Description
https://issues.liferay.com/browse/LHC-729

### Checklist
- [x] Code compiles without errors.
- [ ] Include tests for new features and functionality.
- [ ] Update README/documentation, when applicable.
- [x] All tests, new and existing, pass without errors.
- [ ] Checked browser compatibility, especially IE11.

### CODE REVIEW

File: theme/src/css/_zendesk-variables.scss [hunk 1]
```diff
@@ -114,7 +114,7 @@ $nav-link-padding-x: 1.5rem !default; // 24px
 $nav-underline-link-highlight-left: 0 !default; 
 $nav-underline-link-highlight-right: 1.5rem !default; // 24px
 
-$navbar-height: 4rem !default;
+$navbar-height: 3.5rem !default;
 $navbar-padding-x: 0 !default;
 
 /* ---------- Custom variables ---------- */
```
Comment (jonmak08) [inline theme/src/css/_zendesk-variables.scss]: Can we change all the comments to using the SASS comments if that is the desired format for consistency?

File: theme/src/resources/templates/header.hbs [hunk 2]
```diff
@@ -751,7 +751,7 @@ height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
 		const onNavScroll = function() {
 			const header = document.getElementById('header');
 
-			if (window.scrollY >= 64) {
+			if (window.scrollY >= 56) {
```
Comment (jonmak08) [inline theme/src/resources/templates/header.hbs]: This value seems like an arbitrary number.  Can we set it to a descriptive variable so we know what it is for?

File: theme/src/resources/templates/home_page.hbs [hunk 3]
```diff
@@ -352,6 +352,14 @@
 			}
 		}
 
+		const homepage = document.querySelector('.homepage');
+
+		if (homepage) {
+			const header = document.getElementById('header');
+
+			header.classList.add('hide-header');
```
Comment (jonmak08) [inline theme/src/resources/templates/home_page.hbs]: You should check to make sure the `header` DOM node is returned before doing anything with it.

File: theme/src/css/_zendesk-variables.scss (hunk 1 above)
Comment (rdai10) [reply theme/src/css/_zendesk-variables.scss]: Oops, missed those!

**Comment by rdai10** [comment]
https://github.com/rdai10/zendesk/pull/133 Let me know if you have any other comments! Thanks!

**Pull request merged**